contrib/build/linux/_build.sh

```diff
@@ -4,22 +4,43 @@
 
 set -e  # Exit on error
 
-if [ -z "$1" ] || [ -z "$2" ]; then
-    echo "Please pass Fulcrum and rocksdb dirnames as the two args"
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+    echo "Please pass Fulcrum, rocksdb, and jemalloc dirnames as the three args"
     exit 1
 fi
 
 PACKAGE="$1"
 ROCKSDB_PACKAGE="$2"
+JEMALLOC_PACKAGE="$3"
 TARGET_BINARY=Fulcrum
 
 top=/work
 cd "$top" || fail "Could not cd $top"
 . "$top/$PACKAGE/contrib/build/common/common.sh" || (echo "Cannot source common.h" && exit 1)
 
+info "Running configure for jemalloc ..."
+cd "$JEMALLOC_PACKAGE" || fail "Could not change dir to $JEMALLOC_PACKAGE"
+./autogen.sh -with-jemalloc-prefix= --disable-shared --enable-static \
```

Comment:
Shouldn't it be `--with-jemalloc-prefix=` with 2 dashes?

Reply:
Oh, this is for Linux, it should already default to no prefix there.

Reply:
Eh -- paranoia pays!

Reply:
Ok fixed. Nice catch.  Weird that it worked anyway.. maybe it accepts either?!

Reply:
Weird if it worked, it does say `--with-jemalloc-prefix` in `configure.ac`